moto/glacier/responses.py

```diff
@@ -62,6 +62,7 @@ def _vault_response_delete(self, vault_name: str, headers: Any) -> TYPE_RESPONSE
     def vault_archive_response(
         self, request: Any, full_url: str, headers: Any
     ) -> TYPE_RESPONSE:
+        self.setup_class(request, full_url, headers, use_raw_body=True)
```

Comment:
This was missing here, I assume it never occurred, since there are other calls using glacier and setting up the class before. Now it is a new one, which leads to this new line.